### DISCUSSION TITLE
Is there a way to get serpl to stop leaving a `serpl.log` file in every dir its opened in?

### DISCUSSION THREAD

**Discussion by gwenwindflower** [yassinebridi/serpl#13] [Q&A] [answered]
This tool is so great! Really love the interface design, fantastic work!

One question, I searched the README and couldn't find an answer. It seems to leave this `serpl.log` file behind every time I exit it. Is there a way to make that not happen? 

I'm using MacOS with fish shell in case that's relevant to this behavior. Let me know if you need other details.

**Comment by yassinebridi** [comment, marked as answer]
Sorry about that, i just pushed a fix in the latest version(v0.1.30), please install that.
Thanks for the heads up.
